src/view/components/wrappedField.jsx

```diff
@@ -41,30 +41,21 @@ const getNestedValue = (obj, path) => {
   return obj;
 };
 
-const addDataElementToken = (value, dataElementToken) =>
-  `${value || ""}${dataElementToken}`;
-
 const noop = () => {};
 
 export class DecoratedInput extends React.Component {
-  openDataElementSelector = (tokenize = false) => () => {
+  openDataElementSelector = () => {
```

Comment:
The code changes in this file was just to simplify some things. The data element handling code was copied from another repo and we'll never use one of the features that was supported, so I removed the respective logic.